sql/wsrep_high_priority_service.cc

```diff
@@ -809,6 +810,13 @@ int Wsrep_applier_service::apply_nbo_begin(const wsrep::ws_meta &ws_meta,
     Avoid over-writting of this XID by MySQL XID */
     thd->get_transaction()->xid_state()->get_xid()->set_keep_wsrep_xid(true);
 
+    DBUG_EXECUTE_IF("nbo_stop_before_apply_events", {
+      const char act[] =
+          "now signal nbo_before_apply_events_reached wait_for "
+          "nbo_before_apply_events_continue";
+      assert(!debug_sync_set_action(current_thd, STRING_WITH_LEN(act)));
```

Comment:
Can it be done using just the `thd`?
```suggestion
      assert(!debug_sync_set_action(thd, STRING_WITH_LEN(act)));
```

Reply:
done